common/lib/xmodule/xmodule/mako_module.py

```diff
@@ -17,11 +21,11 @@ class MakoModuleDescriptor(XModuleDescriptor):
     with the name of the template to render, and it will pass
     the descriptor as the `module` parameter to that template
 
-    MakoModuleDescriptor.__init__ takes the same arguments as xmodule.x_module:XModuleDescriptor.__init__
+    MakoTemplateBlock.__init__ takes the same arguments as xmodule.x_module:XModuleDescriptor.__init__
     """
 
     def __init__(self, *args, **kwargs):
-        super(MakoModuleDescriptor, self).__init__(*args, **kwargs)
+        super(MakoTemplateBlock, self).__init__(*args, **kwargs)
```

Comment:
Yes. `MakoTemplateBlock` is a base-class, not a standalone block. I'll change the class name to make that clearer.